src/class-wp-import.php

```diff
@@ -99,6 +99,11 @@ public function import( $file, $options = array() ) {
 
 		$this->options = apply_filters( 'wp_import_options', $options );
 
+		add_filter( 'import_post_meta_key', array( $this, 'is_valid_meta_key' ) );
+		add_filter( 'http_request_timeout', array( &$this, 'bump_request_timeout' ) );
```

Comment:
Let's `remove_filter`, to be good neighbors.

Reply:
This merely moves the add_filter call from below so I'll leave `remove_filter` out of this PR